### PR TITLE
feat(annotations): add region annotation icon

### PR DESCRIPTION
**Pull request by mickr**
Adding the region annotation icon

### CODE REVIEW

File: src/icons/annotations/IconRegionAnnotation.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { white } from '../../styles/variables';
+
```
Comment (ConradJChan) [inline src/icons/annotations/IconRegionAnnotation.tsx:3]: Whitespace needed? Also 🔤

File: src/icons/annotations/IconRegionAnnotation.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import AccessibleSVG from '../accessible-svg';
+
+import { Icon } from '../iconTypes';
+
+const IconRegionAnnotation = ({ className = '', color = white, height = 21, title, width = 22 }: Icon) => (
```
Comment (ConradJChan) [inline src/icons/annotations/IconRegionAnnotation.tsx:8]: Are these dimensions correct? Looks like preview SDK has different dimensions

File: src/icons/annotations/IconRegionAnnotation.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    <AccessibleSVG
+        className={`icon-annotation-region ${className}`}
```
Comment (ConradJChan) [inline src/icons/annotations/IconRegionAnnotation.tsx:10]: Should `icon-annotation-region` -> `icon-region-annotation` to be consistent?

Comment (mickr) [reply src/icons/annotations/IconRegionAnnotation.tsx:10]: I followed the other existing classnames

File: src/icons/annotations/IconRegionAnnotation.tsx (hunk 2 above)
Comment (mickr) [reply src/icons/annotations/IconRegionAnnotation.tsx:8]: It looked a little small with the smaller dimensions

File: src/icons/annotations/IconRegionAnnotation.tsx (hunk 3 above)
Comment (ConradJChan) [reply src/icons/annotations/IconRegionAnnotation.tsx:10]: I realized we may also want to do SUIT styles, `bdl-IconRegionAnnotation`

File: src/icons/annotations/IconRegionAnnotation.js.flow [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import * as React from 'react';
+import { white } from '../../styles/variables';
+
+import AccessibleSVG from '../accessible-svg';
+
+import type { Icon } from '../flowTypes';
+
+const IconRegionAnnotation = ({ className = '', color = white, height = 21, title, width = 22 }: Icon) => (
+    <AccessibleSVG
+        className={`icon-annotation-region ${className}`}
```
Comment (ConradJChan) [inline src/icons/annotations/IconRegionAnnotation.js.flow:11]: Not sure if it matters for `.flow` files, but `bdl-IconRegionAnnotation` for consistency?

**Pull request merged**